Docs/articles/Kernel/VFS.md

```diff
@@ -3,56 +3,73 @@
 In this article we will discuss about using Cosmos VFS (virtual file system).
 Cosmos VFS and the VFS manager classes, let you manage your file system.
 
+**Attention**: **Always** format your drive with Cosmos and **only** Cosmos if you plan to use it with Cosmos. Using any other tool such as Parted, FDisk (or any other tool) might lead to weird things when using that drive with Cosmos' VFS. Those tools are much more advanced and might format and read/write to the disk differently than Cosmos.
+
 First, we should create and initialize an instance of the VFS, this will initialize the partition and files-system lists, as well as register the new VFS.
 This is essential for using the VFS.
 
-We start with creating global CosmosVFS, this line should appear outside of any function, and before the BeforeRun() function.  
+We start with creating a global CosmosVFS, this line should appear outside of any function, and before the BeforeRun() function.
 
 ```C#
-CosmosVFS fs = new Sys.FileSystem.CosmosVFS();
+var fs = new Sys.FileSystem.CosmosVFS();
 ```
 
-Next, we register our VFS at the VFS manager, this will initiate the VFS and make it usable, add this to your kernel's BeforeRun():
+Next, we register our VFS at the VFS manager, this will initiate the VFS and make it usable, add this to your kernel's BeforeRun() function:
 
 ```C#
 Sys.FileSystem.VFS.VFSManager.RegisterVFS(fs);
 ```
 
-After the initialization process is done, a message like this would apper on your screen:
+After the initialization process is done, a message like this would appear on your screen:
 ![Initialize](https://raw.githubusercontent.com/CosmosOS/Cosmos/master/Docs/articles/Kernel/images/File%20System%20Initialize.PNG)
 
-This message is printed by the register method and it provides info about the file system.
+This message is printed by the RegisterVFS() method and it provides info about the partition.
+
+After our VFS has been initialized, we can use more interesting functions, let's go over some of them:
+
+## Format drive
+
+**Note**: You don't have to format your drive if you're debugging your Cosmos project with VMWare. The build will automatically add an already formatted FAT32 VMDK file for your convenience.
+
+You can format your drive with the Format() function, just like this:
```

Comment:
Please add that Cosmos can only format the drive that already has FAT32.

Reply:
Really? Well I didn't know that. I'll add that.